packages/frontend/amp/components/Byline.tsx

```diff
@@ -1,31 +1,37 @@
 import React from 'react';
 import { bylineTokens } from '@frontend/amp/lib/byline-tokens';
 import { css } from 'emotion';
-import { headline } from '@guardian/pasteup/typography';
-import { pillarPalette } from '@frontend/lib/pillars';
-
-const bylineStyle = (pillar: Pillar) => css`
-    ${headline(2)};
-    color: ${pillarPalette[pillar].main};
-    padding-bottom: 8px;
-    font-style: italic;
-
-    a {
-        font-weight: 700;
-        color: ${pillarPalette[pillar].main};
-        text-decoration: none;
-        font-style: normal;
+
+const profileImageStyle = css``;
+
+const profileImage = (contributorTag: TagType) => {
```

Comment:
This currently does nothing (as no Frontend data is passed) so I might remove this here for now.

Reply:
I don't mind. Maybe add a TODO comment pointing to the Trello card or something so we don't lose sight of it?